### PR TITLE
Allow command server to download new needles on resume

### PR DESCRIPTION
**Pull request by Martchus**
see https://progress.opensuse.org/issues/41075  
os-autoinst: https://github.com/os-autoinst/os-autoinst/pull/1027

### CODE REVIEW

File: t/ui/07-file.t [hunk 1]
```diff
@@ -25,6 +25,7 @@ use lib "$FindBin::Bin/../lib";
 use Test::More;
 use Test::Mojo;
 use Test::Warnings;
+use File::Path qw(make_path remove_tree);
```
Comment (mudler) [inline t/ui/07-file.t]: Why not using Mojo::File ?

Comment (Martchus) [reply t/ui/07-file.t]: File::Touch ist used for mtime in os-autoinst. If there's a better way to achieve the same I'm open for suggestions.

File::Path is already used elsewhere and hence present anyways. So why not using it here, too?

Comment (mudler) [reply t/ui/07-file.t]: For one simple reason: consistency See discussion in https://github.com/os-autoinst/os-autoinst/pull/1006#discussion_r207259933

**Comment by Martchus** [comment]
I changed it to use `Mojo::File` and it works on staging (https://e212.suse.de/tests/5). So I guess we can merge this when the tests are passing. Then the docker image can be rebuild to include `File::Touch` and tests on the os-autoinst side have a change to pass as well.

**Comment by Martchus** [comment]
I've just added a limit on the openQA-side, too (to address https://github.com/os-autoinst/os-autoinst/pull/1027#issuecomment-424024745). When the tests still pass after rebasing and that modification the PR can be merged in my opinion.

**Pull request merged**